e2e/tests/utils/setup.ts

```diff
@@ -2,14 +2,21 @@ import { clearDB } from "@graphql-debugger/data-access";
 
 import * as backend from "./backend";
 
+const shouldSpawnBacked = process.env.E2E_IN_DOCKER !== "true";
+
 beforeAll(async () => {
-  await backend.listen();
+  if (shouldSpawnBacked) {
+    await backend.listen();
+  }
 });
 
 beforeEach(async () => {
+  // TODO - for docker we might have to add an api to clear the db as this will point to the host machine
```

Comment:
clearDB could be talking to a different dev.db to the docker instance.

Reply:
Yeah this should hit an endpoint (?) exposed on the `Docker` container if we're haven't spawned a backend process locally - some kind of dev only endpoint to nuke the DB.

Reply:
`/api/v1/clear-db`

Reply:
or it can be added thru the graphql-api as a mutation. But using the collectors port for this might be the best simplest option.